### ISSUE TITLE
Maintenance status + sustainability

### ISSUE THREAD

**Issue by ml-evs** [echemdata/galvani#91]
Thanks for all the suggestions + data sharing here! I think this shows how widely used galvani still is, so we should put in some effort to ensure its ongoing maintenance. Does anyone have a line on @chatcannon so we could potentially get some maintainers added to the repo? (I guess naturally that means I volunteer but would prefer not to do it alone...) To avoid hijacking this issue I will raise a new one based on this comment, but just wanted to encourage people from this discussion to think about it!

_Originally posted by @ml-evs in https://github.com/echemdata/galvani/issues/87#issuecomment-1757469899_

**Comment by chatcannon** [comment]
You're right, I haven't had much time to pay attention to my GitHub over the past year and won't have much time for the immediate future either.

**Comment by ml-evs** [comment]
Thanks for the invite @chatcannon and totally understand. I won't have time to contribute new features or much else beyond basic maintenance, but hopefully at least this will allow others to contribute more easily.

Would be great to get a couple more people involved to share the load!

**Comment by ml-evs** [comment]
I guess the most immediate thing to do would be also managing future releases -- @chatcannon would you be happy for me to be added as a collaborator on PyPI too? My username is the same there. I would suggest then setting up GH actions to automatically push a release to PyPI for every GitHub release.

**Comment by chatcannon** [comment]
I will hold on to PyPI for the time being as it's harder to undo things that are done there, and also the last time I added someone as a maintainer to echemdata he stopped doing anything after a few months.

**Comment by ml-evs** [comment]
As you prefer -- I wonder if you would mind giving me a back channel to contact you on then? Can guarantee to only use for very special occasions like making a release 😁 My contact details are found at the bottom of my personal site linked in my GitHub profile.

We are probably due a release now -- I'd rather not start messing things up until we're on a stable base.

**Comment by ml-evs** [comment]
Just wanted to follow this up @chatcannon, any chance we could get a GitHub + PyPI release before we start tackling #87 etc?

**Comment by chatcannon** [comment]
I just released 0.3.0

I didn't realise the last release was three years ago - I was sure I had done one more recently than that. Oh well.

**Comment by ml-evs** [comment]
Can we get another release please @chatcannon? The LFS quota issue makes it a bit tricky to install this package with `pip` via git directly (and indeed uses up more bandwidth). Only the PyPI release can solve this. Again, I'd be happy to help automate this process (so that a PyPI release is made when a GitHub release is created).

**Comment by ml-evs** [comment]
Right now my permissions are a bit weird in that I can't merge PRs into `master` nor release to PyPI but I can make GitHub releases...

**Comment by chatcannon** [comment]
@ml-evs I changed your permissions from "Write" to "Maintain" - let's see if things are better now.
